new_course/1_programming_fundementals/methods.md

```diff
@@ -73,27 +73,27 @@ Your method names can use numbers, capital letters, lowercase letters, and the s
 
 Here are some things you are not allowed to do with your method names:
 
-* You cannot name your method one of Ruby's approximately 40 reserved words, such `end`, `while`, or `for`. Do a quick scan of the [full list here](http://www.java2s.com/Code/Ruby/Language-Basics/Rubysreservedwords.htm).
+* You cannot name your method one of Ruby's approximately 40 reserved words, such as `end`, `while`, or `for`. Checkout the full list [here](http://www.java2s.com/Code/Ruby/Language-Basics/Rubysreservedwords.htm).
 * You cannot use any symbols other than `_`, `?`, `!`, and `=`.
 * You cannot use `?`, `!`, or `=` anywhere other than at the end of the name.
 * You cannot begin a method name with a number.
 
 Here are some examples of valid and invalid method names:
 
-~~~ruby
-method_name     # valid
-_name_of_method # valid
-1_method_name    # invalid
-method_27       # valid
-method?_name    # invalid
-method_name!    # valid
-begin           # invalid  (Ruby reserved word)
-begin_count     # valid
+~~~
```

Comment:
We need ruby after the `~~~` for our markdown parse on the main site to be able to format code blocks correctly. Can you add that back in here please? 😄

Reply:
Interesting. I had taken out `ruby` because it was syntax highlighting the names, which I thought was a bit confusing (names appeared in blue, black, red or a combination of colors). But I can definitely put it back as soon as I get back on my computer.

Reply:
ah I see, in that case I'd leave it as is. It will just format as standard uniform code. This was a good change. Thanks @leonardovega

Reply:
Thanks @KevinMulhern for guiding me through this process. I know you and the rest of the TOP team are very busy. So I appreciate you guys making the contribution process as welcoming as possible.